test/abort.js

```diff
@@ -1,90 +1,45 @@
 
-var mux  = require('../')
-var tape = require('tape')
-var pull = require('pull-stream')
-var Abortable = require('pull-abortable')
+const mux = require('../')
+const tape = require('tape')
+const pull = require('pull-stream')
+const Abortable = require('pull-abortable')
 
 function id (e) { return e }
 
 module.exports = function (serializer) {
   tape('stream abort', function (t) {
-    t.plan(2)
-
-    var client = {
+    t.plan(3)
+    const abortable = Abortable()
+    const client = {
       drainAbort: 'sink'
     }
 
-    var A = mux(client, null, serializer) ()
```

Comment:
Great, this test was really strange before, looked like some copy paste that was never cleaned?

Reply:
yeah, really strange pattern of having **two** module.exports assignments